packages/test/test-end-to-end-tests/src/test/summarizeRestart.spec.ts

```diff
@@ -33,6 +33,7 @@ describeNoCompat("Summarizer closes instead of refreshing", (getTestObjectProvid
 	beforeEach(async () => {
 		provider = getTestObjectProvider({ syncSummarizer: true });
 		settings["Fluid.ContainerRuntime.Test.SummarizationRecoveryMethod"] = "restart";
+		settings["Fluid.ContainerRuntime.Test.CloseSummarizerOnSummaryStaleDelayOverrideMs"] = 0; // so that we don't have to wait for the timeout
```

Comment:
I'd let it wait. As in - test what we intend to ship.
Are you afraid that our service tests generate so many ops that with delay we will not catch up?

Reply:
No, not really, this was more for convience of local testing than anything, I can add the wait back